site/docs/sql/intro.md

```diff
@@ -1,351 +1,399 @@
 
-When you use the distribution package, the `hazelcast-jet-sql` is on
-the class path by default. Nevertheless, make sure to have the
-`hazelcast-jet-sql-{jet-version}.jar` file in the `lib/`
-directory.
-
-If you use Jet in embedded mode, besides the `hazelcast-jet` dependency
-add also the `hazelcast-jet-sql` dep:
-
-<!--DOCUSAURUS_CODE_TABS-->
-
-<!--Gradle-->
-
-```groovy
-compile 'com.hazelcast.jet:hazelcast-jet-sql:{jet-version}'
+```text
+wget https://github.com/hazelcast/hazelcast-jet/releases/download/v4.4/hazelcast-jet-4.4.tar.gz
+tar xvf hazelcast-jet-4.4.tar.gz
+cd hazelcast-jet-4.4
+bin/jet-start
 ```
 
-<!--Maven-->
+You are now ready to write some Jet SQL. Start another terminal window
+and try this:
 
-```xml
-<dependency>
-    <groupId>com.hazelcast.jet</groupId>
-    <artifactId>hazelcast-jet-sql</artifactId>
-    <version>{jet-version}</version>
-</dependency>
+```sql
+$ bin/jet sql
+Connected to Hazelcast Jet 4.4 at [192.168.5.13]:5701 (+0 more)
+Type 'help' for instructions
+sql> select * from TABLE(generate_series(1,5));
++------------+
+|           v|
++------------+
+|           1|
+|           2|
+|           3|
+|           4|
+|           5|
++------------+
+5 row(s) selected
+sql> select sum(v) from TABLE(generate_series(1,5));
++--------------------+
+|              EXPR$0|
++--------------------+
+|                  15|
++--------------------+
+1 row(s) selected
+sql>
 ```
 
-<!--END_DOCUSAURUS_CODE_TABS-->
+Here are two more examples with streaming SQL. Streaming queries never
+complete, so use `Ctrl-C` to cancel them after a while:
 
-If you use other features from SQL such as the Kafka or File connectors,
-also add the `hazelcast-jet-kafka` or `hazelcast-jet-hadoop` modules in
-the same way.
+```sql
+sql> SELECT * FROM TABLE(generate_stream(10));
++--------------------+
+|                   v|
++--------------------+
+|                   0|
+|                   1|
+|                   2|
+|                   3|
+|                   4|
+|                   5|
+^C
+Query cancelled.
+sql> SELECT * FROM TABLE(generate_stream(100)) WHERE v / 10 * 10 = v;
++--------------------+
+|                   v|
++--------------------+
+|                   0|
+|                  10|
+|                  20|
+|                  30|
+|                  40|
+|                  50|
+^C
+Query cancelled.
+sql>
+```
 
-## Example
+## Features in this Beta
 
-In this example we'll show how to query Apache Kafka using SQL and
-stream the messages to an IMap.
+In this beta release, you can use these:
 
-First, make sure you have Apache Kafka up and running. You can follow
-the [instructions here](https://kafka.apache.org/quickstart). Start the
-broker and create a topic named `trades`.
+- [SELECT and WHERE
+expressions](https://docs.hazelcast.org/docs/{imdg-version}/manual/html-single/index.html#expressions)
+- FROM [Apache Kafka topics](kafka-connector.md) and
+[files (local and remote)](file-connector.md)
```

Comment:
This is the pain point of two documentations. You can also do `FROM imaps`, but that will be executed by imdg - you can't use aggregate functions or join two maps. There's also a lot more documentation in imdg like supported types, functions, operators...